--- a/Makefile
+++ b/Makefile
@@ -34,21 +34,21 @@
 	@echo "\n-------------enter isp install------------------------->"
 	@install -d ${INSTALL_DIR}${libdir}
 	@install -d ${INSTALL_DIR}${includedir}
-	@install -d ${INSTALL_DIR}/opt/imx8-isp/bin
-	@install -d ${INSTALL_DIR}/opt/imx8-isp/bin/dewarp_config
-	@install -m 644 -D $(XML) ${INSTALL_DIR}/opt/imx8-isp/bin
-	@install -m 755 -D $(TUNINGEXT) ${INSTALL_DIR}/opt/imx8-isp/bin
-	@install -m 755 -D $(DRV) ${INSTALL_DIR}/opt/imx8-isp/bin
-	@install -m 755 -D $(MEDIA_SERVER) ${INSTALL_DIR}/opt/imx8-isp/bin
-	@install -m 755 -D $(VVEXT) ${INSTALL_DIR}/opt/imx8-isp/bin
-	@install -m 755 -D $(VIDEO_TEST) ${INSTALL_DIR}/opt/imx8-isp/bin
-	@install -m 755 -D $(SCRIPT) ${INSTALL_DIR}/opt/imx8-isp/bin
+	@install -d ${INSTALL_DIR}$(libdir)/imx8-isp
+	@install -d ${INSTALL_DIR}$(libdir)/imx8-isp/dewarp_config
+	@install -m 644 -D $(XML) ${INSTALL_DIR}$(libdir)/imx8-isp
+	@install -m 755 -D $(TUNINGEXT) ${INSTALL_DIR}$(libdir)/imx8-isp
+	@install -m 755 -D $(DRV) ${INSTALL_DIR}$(libdir)/imx8-isp
+	@install -m 755 -D $(MEDIA_SERVER) ${INSTALL_DIR}$(libdir)/imx8-isp
+	@install -m 755 -D $(VVEXT) ${INSTALL_DIR}$(libdir)/imx8-isp
+	@install -m 755 -D $(VIDEO_TEST) ${INSTALL_DIR}$(libdir)/imx8-isp
+	@install -m 755 -D $(SCRIPT) ${INSTALL_DIR}$(libdir)/imx8-isp
 
 	@if [ -d $(${INSTALL_DIR}${includedir})]; \
 		then cp -av --no-preserve=ownership $(INCLUDE) ${INSTALL_DIR}${includedir}; \
 	fi;
-	@if [ -d $(${INSTALL_DIR}/opt/imx8-isp/bin/dewarp_config)]; \
-		then cp -av --no-preserve=ownership $(DEWARP_CONFIG) ${INSTALL_DIR}/opt/imx8-isp/bin/dewarp_config; \
+	@if [ -d $(${INSTALL_DIR}$(libdir)/imx8-isp/dewarp_config)]; \
+		then cp -av --no-preserve=ownership $(DEWARP_CONFIG) ${INSTALL_DIR}$(libdir)/imx8-isp/dewarp_config; \
 	fi;
 	@if [ -d $(${INSTALL_DIR}${libdir})]; \
 		then cp -av --no-preserve=ownership $(LIBS) ${INSTALL_DIR}${libdir}; \
