From 81079ca4f187c3c02fa96349e87ac52ac19e186b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=A9bastien=20Taylor?= <sebastien@au-zone.com>
Date: Thu, 9 Nov 2023 15:26:07 -0700
Subject: [PATCH] Use local bno08x-rs repository.

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 5d5189a..a6ce4b7 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -18,4 +18,4 @@ clap = { version = "4.4.6", features = ["derive"] }
 zenoh = "0.10.0-rc"
 zenoh-ros-type = { git = "https://github.com/MaivinAI/zenoh-ros-type" }
 cdr = "0.2.4"
-bno08x = { git = "ssh://git@bitbucket.org/au-zone/bno08x-rs", rev = "b614b976" }
+bno08x = { path = "../bno08x-rs" }
--
2.25.1

